mappings/net/minecraft/world/biome/BiomeSpecialEffects.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_4763 net/minecraft/world/biome/BiomeSpecialEffects
```

Comment:
Whats so special? Why not BiomeEffects?

Reply:
moj call them special effects. yep should just use effects, will fix

Reply:
well they're specifically visual effects, which are often called special effects.

Reply:
I will just go with `effects`. Don't think fog color or water color is that "special".